File: 1. Fundamentos de Programación para Desarrollo Full Stack/Java/condicionales.java
// Crea un programa que determine si un número está en el rango de 1 a 100.

import java.util.Scanner;

public class condicionales {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Introduce un número: ");
        int numero = sc.nextInt();
        sc.close();
        if (numero >= 1 && numero <= 100) {
            System.out.println("El número está en el rango de 1 a 100");
        } else {
            System.out.println("El número no está en el rango de 1 a 100");
        }
    }
}

File: 1. Fundamentos de Programación para Desarrollo Full Stack/Python/condicionales.py
# Escribe un programa que indique si un número es par o impar.


def par_o_impar(n):
    if n % 2 == 0:
        return f"El número {n} es par"
    else:
        return f"El número {n} es impar"


print(par_o_impar(2))
print(par_o_impar(3))
print(par_o_impar(4))
print(par_o_impar(5))
